google_maps/static/css/google_maps_edit.css

```diff
@@ -0,0 +1 @@
+/* CSS for GoogleMapsXBlock studio view */
```

Comment:
I'm not much of a fan of adding empty files, makes more sense to me to add something once you have something to add. But I'm nitpicking here.